gemini.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/zsh 
+#!/bin/bash 
 # Copyright (C) 2001-2020  Alex Schroeder <alex@gnu.org>
 # Copyright (C) 2022 Tristan Havelick (tristan@havelick.com) 
-#  for modifications to work with zsh
+#  for modifications to work with bash
```

Comment:
This comment doesn't make sense anymore let's kill it. Update the year for my copyright to 2025